minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
If its going to go into the API, I think the following needs to be considered.

  * Need a getter in the API.  What should the getter do/return if not set?
  * Do we want to allow the user to clear this setting, if so how does the user clear this setting?
  * Need to document in javadoc its relationship to `setZooKeeperPort()`.  Should these be made mutually exclusive?  Trying to set both would cause the 2nd set to throw an exception?